boards/synopsys/em_starterkit/em_starterkit_emsk_em9d.dts

```diff
@@ -8,7 +8,7 @@
 
 #include <mem.h>
 #include <synopsys/emsk.dtsi>
-#include "em_starterkit_r23.dtsi"
+#include "em_starterkit_r22.dtsi"
```

Comment:
why  "_r22" instead of "_r23"?

Reply:
> The original board files were wrong, the documentation states that em11d is supported on r2.3 and r2.2 but actually did not allow selecting r2.2

With these changes, the boards can correctly be changed between r2.2 and r2.3

Reply:
Removed